knora/dsplib/utils/excel_to_json_lists.py

```diff
@@ -268,136 +260,81 @@ def simplify_name(value: str) -> str:
     return simplified_value
 
 
-def _make_root_node_from_args(
```

Comment:
Here, GitHub's diff visualizer messes up :-)
I removed `_make_root_node_from_args()` entirely, and modified `validate_lists_section_with_schema()`.